nab-starter/src/test/java/ru/hh/nab/starter/server/jetty/HHServerConnectorGracefulTest.java

```diff
@@ -43,33 +39,6 @@ public static void afterGracefulServletContextHandlerTestClass() {
     executorService.shutdown();
   }
 
-  @Test
-  public void testOriginalServerConnectorDoesNotWaitPendingRequests() throws Exception {
```

Comment:
?

Reply:
https://github.com/hhru/nuts-and-bolts/pull/414#discussion_r1254159075